docs/source/data/errors.mdx

```diff
@@ -482,3 +482,36 @@ In this case, the error above is reported to Apollo Studio as:
 ```
 The REDACTED doesn't have sufficient privileges.
 ```
+
+## HTTP Status Codes
+
+GraphQL, by design, does not use the same conventions from REST to communicate via HTTP verbs and status codes.
+Schema implementors should put information the clients need in the schema, or as part of the standard response `errors` field.
+If possible, consider using the included [Error Codes](#error-codes) or [Custom Errors](#custom-errors) first before modifying http fields.
+
+If you need to have custom behavior, you can set the `http` fields on the response via a [plugin](../integrations/plugins).
+Be aware that GraphQL client libraries may not treat all response status codes the same, and so it will be up to your team to decide what patterns are used.
+
+As an example, here is how you might set a response header and status code based off the GraphQL errors:
```

Comment:
Slight tweak recommend here: "As an example, here is how you could set a custom response header and status code based on a GraphQL error:"